### PR TITLE
Use rasp and fix data handler

### PR DESCRIPTION
**Pull request by dlrobertson**
- Use rasp for http goodness
 - Punt the old data handler
   - Create a data handler thread that bulk inserts at some interval
   - The worker threads now just post their retrieved pages to this
     list of pages to be bulk inserted with `executemany`
   - Rely on `psycopg2`s encoding. Don't try to hand format the
     insert string.
   - Remove MongoDB stuff
   - Remove wierd exceptions
 - Remove old codez
 - Re-License to a more liberal license (BSD)

This is far from complete, but this removes a lot of old stuff that
either doesn't work or is super fragile. After this 60% of the time it
*should* work all of the time.

### CODE REVIEW

**Comment by dlrobertson** [comment]
Thanks @baronvonbadguy and @wdm0006 for making this possible.

File: example.py [hunk 1]
```diff
@@ -1,52 +1,43 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2017 Anidata
```
Comment (bmenn) [inline example.py:1]: Not important right now, but I think we need to pick a license and stick to it and centralize in the near future.

**Review by bmenn — CHANGES_REQUESTED** [review]
I think we should add `anidata/rasp` to `setup.py`, but otherwise lgtm.

**Review by bmenn — APPROVED** [review]

**Comment by bmenn** [comment]
Made the change requested here in #12.

**Pull request merged**